.github/workflows/openssl.yml

```diff
@@ -1,6 +1,21 @@
 name: OpenSSL CI
 
-on: [push, pull_request]
+on:
+  push:
+    paths:
+      - 'src/openssl/**'
+      - 'spec/std/digest/**'
+      - 'spec/std/openssl/**'
+      - '.github/workflows/openssl.yml'
+  pull_request:
+    paths:
+      - 'src/openssl/**'
+      - 'spec/std/digest/**'
+      - 'spec/std/openssl/**'
+      - '.github/workflows/openssl.yml'
+  schedule:
+    - cron: '0 3 * * *'
```

Comment:
Perhaps it would be a good idea to set the schedule on bit different time for each of the workflows to avoid CI congestion  and potentially speed up the runs.

Reply:
I believe we have had this discussion before. But I don't recall where.

Congestion isn't that bad. These workflows are now running only once per day, not on every commit (often even twice per).

Speed up isn't really relevant here. We just want them to run at some point in a regular interval. Doesn't matter much when exactly.

In fact, I'm considering to introduce a central orchestration workflow which runs on a schedule and calls the other ones (mentioned in https://github.com/crystal-lang/crystal/issues/14983#issuecomment-2644182192). At that point all workflows would be triggered at the same time anyway.